sympy/calculus/util.py

```diff
@@ -61,15 +63,18 @@ def continuous_domain(f, symbol, domain):
     try:
         sings = S.EmptySet
         if f.has(Abs):
-            sings = solveset(1/f, symbol, domain)
+            sings = solveset(1/f, symbol, domain) + \
+                solveset(denom(together(f)), symbol, S.Reals)
```

Comment:
Should this be solved in `S.Reals` or `domain`?

Reply:
Yeah, it should be `domain` instead. Updated the PR.